core/src/main/java/org/testcontainers/core/CreateContainerCmdModifier.java

```diff
@@ -0,0 +1,14 @@
+package org.testcontainers.core;
+
+import com.github.dockerjava.api.command.CreateContainerCmd;
+
+/**
+ * Callback interface that can be used to customize a {@link CreateContainerCmd}.
+ */
+public interface CreateContainerCmdModifier {
+    /**
+     * Callback to modify a {@link CreateContainerCmd} instance.
+     * @param createContainerCmd the create command to modify
+     */
+    void modify(CreateContainerCmd createContainerCmd);
```

Comment:
Thinking out loud: we probably want to have it as a `Function<CreateContainerCmd, CreateContainerCmd>` if we consider making commands immutable in the future (forward compatibility) - WDYT?

Reply:
I've applied the change.